plugins/MachineSettingsAction/MachineSettingsAction.qml

```diff
@@ -164,7 +164,7 @@ Cura.MachineAction
                                 id: gcodeFlavorComboBox
                                 sourceComponent: comboBoxWithOptions
                                 property string settingKey: "machine_gcode_flavor"
-                                property string label: catalog.i18nc("@label", "GCode flavor")
+                                property string label: catalog.i18nc("@label", "Gcode flavor")
```

Comment:
I don't disagree with you. I actually had all occurrences changed to "G-code" locally, but then noticed fdmprinter.def.json (where most of the tooltips come from) is (mostly) standardised in "Gcode". I did not want to make changes to fdmprinter.def.json for this PR.

Both RepRapWiki and Wikipedia use "G-code".

Your greps are probably influenced by i18n files, and to non-userfacing text and variable names.